CorpusSearch/ClientApp/src/routes/NewDocList.tsx

```diff
@@ -0,0 +1,40 @@
+/** NewDocList
+ * Provides a list of links to the latest and greatest files in the Corpus
+ * Which means those document.csv files that git detects as having recent changes
+ */
+import React, {useEffect, useState} from "react"
+function CorpusLink({ident="",name=""}): JSX.Element {
+    if ( name.slice(0,2) != "🎥"){
+        name = "📖" + name
+    }
+    return <div>
+        <a href={"docs/" + ident}>{name}</a>
+    </div>
+}
+export const NewDocList = () => {
+type DocType =  {name:string, ident:string, uploaded:string}
+const [newDocs, setNewDocs  ] 
+    = useState<DocType[]>([])
+useEffect(()=>  {
+    const fetchNewDocs = async () => {
+        await fetch("api/metadata/latest") .then((res)=> {
+            if (!res.ok) console.warn(`fetch api/metadata/latest returned: ${res.status}`)
+            return res.json()
+        }).then((json: DocType[])=> { setNewDocs(json) })
+    }
+    fetchNewDocs().catch((e:Error)=> console.warn(`fetchNewDocs returned:" ${e.message}`))
+},[]
+    )
+    const len = newDocs.length
+    if (!len) return <></>
+    return (<div className="NewDocList">
+        <header className="NewDocList-header">
+        </header>
+        <div>Recent Additions and Updates:<br/>
```

Comment:
Don't make people read

Not sure about the bolding - your call

```suggestion
        <div><b>Recent changes:</b><br/>
```